feat(nx-plugin): adjust bin path handling (#1109)
code-pushup/cli: CONTRIBUTING.md

To change this setup, open (or create) the `.env` file in the root folder.
Edit or add the environment variable there as follows: `INCLUDE_SLOW_TESTS=true`.

### Executing local code

> We use the current version of CodePushup and its plugins, to measure itself and its plugins.

Reviewer: Branding :sweat_smile: 

```suggestion
We use the current version of Code PushUp and its plugins, to measure itself and its plugins.
```

_Execute the latest CLI source_

```jsonc
// project.json
{
  "targets": {

>     "exec-local-cli-source": {
>       "executor": "nx:run-commands",
>       "options": {
>         "command": "node packages/cli/src/index.ts",
>         "args": ["--no-progress", "--verbose", "--help"],
>       }
>     },
>     "exec-local-cli-source-and-local-plugin-source": {
>       "executor": "nx:run-commands",
>       "options": {
>         "command": "node packages/cli/src/index.ts",
>         "args": ["--no-progress", "--verbose", "--onlyPlugins=js-packages"],
>         "env": {
>           "NODE_OPTIONS": "--import tsx",
>           "TSX_TSCONFIG_PATH": "tsconfig.base.json"
>         }
>       }

Reviewer: A couple of things:
- the `--help` in `exec-local-cli-source` target would not run any commands, probably not what you want
- `exec-local-cli-source` is missing the `tsx` env vars, so it wouldn't work
- it would be more convenient to have 1 reusable target than 2 specialized targets (won't scale), if you just expose the CLI, then any arguments can be appended (e.g. `-- --onlyPlugins=js-packages`)
- the target names are really long - why not just call the target `code-pushup`? that's what it's always called, and it does the same thing (just from source instead of npm)
 
```suggestion
    "code-pushup": {
      "executor": "nx:run-commands",
      "options": {
        "command": "node packages/cli/src/index.ts",
        "args": ["--no-progress", "--verbose"],
        "env": {
          "NODE_OPTIONS": "--import tsx",
          "TSX_TSCONFIG_PATH": "tsconfig.base.json"
        }
      }
```

    }
 }
```

_Setup code-pushup targets with the nx plugin_

```jsonc
// nx.json
{
  "plugins": [
    {
      "plugin": "@code-pushup/nx-plugin",
      "options": {
        "cliBin": "node ./packages/cli/src/index.ts",
        "env": {

>           // nx.json

Reviewer: The `nx.json` comment is duplicated:

```suggestion
```
